src/webserver/HttpBase.h

```diff
@@ -221,6 +227,9 @@ class HttpBase
         if (!ip)
             return;
 
+        BOOST_LOG_TRIVIAL(info) << "[" << tag() << "] http::" << __func__
+                                << " received request from ip = " << *ip;
```

Comment:
I think info is too high for this. I would also add something like "Posting to WorkQueue".